src/Makefile

```diff
@@ -18,6 +18,7 @@ TARGET_SOURCES_TEST := memcachedstore.cpp \
                        snmp_row.cpp \
                        snmp_accumulator_table.cpp \
                        snmp_counter_table.cpp \
+					   snmp_success_fail_count_table.cpp \
```

Comment:
This looks like it's been tab-indented rather than space-indented - worth checking with @AlexHockey in case he can suggest a .vimrc fix to make Vim use spaces automatically.